modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -73,15 +73,15 @@ You can use an existing account to join an existing organization once you receiv
 * Admin: An account with the organization administrator role associated to a different organization.
 * Owner: An organization owner account for a different organization.
 
-After clicking the link in the invitation, you can choose the account to use to join the organization.
+After clicking the link in the invitation, you can choose the account to use to join the organization. The link expires in one week.
```

Comment:
Do both the link to accept and the ability to choose an account expire in one week, or just the link to accept? Clarify.
```suggestion
After clicking the link in the invitation, you have one week to choose the account to use to join the organization. 
```